src/integrationtest/StatementExecution.test.ts

```diff
@@ -224,26 +226,56 @@ describe("StatementExecution", function() {
         chai.assert.equal(searchCount, 0);
     });
 
-    it("Can return metrics for consumed IOs", async () => {
+    it("Can return metrics for execute()", async () => {
         const struct1: Record<string, string> = {
             [constants.COLUMN_NAME]: constants.MULTI_DOC_VALUE_1
         };
         const struct2: Record<string, string> = {
             [constants.COLUMN_NAME]: constants.MULTI_DOC_VALUE_2
         };
         const insertStatement: string = `INSERT INTO ${constants.TABLE_NAME} <<?,?>>`;
-        let ioUsage: IOUsage = await driver.executeLambda(async (txn: TransactionExecutor) => {
-            return (await txn.execute(insertStatement, struct1, struct2)).getConsumedIOs();
+        await driver.executeLambda(async (txn: TransactionExecutor) => {
+            await txn.execute(insertStatement, struct1, struct2);
         });
-        chai.assert.equal(ioUsage.getReadIOs(), 0);
 
         const searchQuery: string = `SELECT VALUE ${constants.COLUMN_NAME} FROM ${constants.TABLE_NAME}` +
             ` WHERE ${constants.COLUMN_NAME} IS NULL`;
-        ioUsage = await driver.executeLambda(async (txn: TransactionExecutor) => {
-            return (await txn.execute(searchQuery)).getConsumedIOs();
+        const result: Result = await driver.executeLambda(async (txn: TransactionExecutor) => {
+            return await txn.execute(searchQuery);
         });
-        chai.assert.equal(ioUsage.getReadIOs(), 2);
 
+        chai.assert.isNotNull(result.getConsumedIOs());
+        chai.assert.isNotNull(result.getTimingInformation());
+        chai.assert.equal(result.getConsumedIOs().getReadIOs(), 2);
+        chai.assert.isTrue(result.getTimingInformation().getProcessingTimeMilliseconds() > 0);
+    });
+
+    it("Can return metrics for executeAndStreamResults()", async () => {
+        const insertStatement: string = `INSERT INTO ${constants.TABLE_NAME} << {'col': 1}, {'col': 2}, {'col': 3} >>`;
```

Comment:
can we make this into a helper function and have `Can return metrics for execute()` test call this so they both are querying the same data?

Reply:
Combined the 2 to be inline with other drivers